Sunburst37/ASSEMBLY_AND_OPERATION_INFORMATION.agc

```diff
@@ -84,20 +70,20 @@
 # MP 7 - SIVB/LEM SEPARATION
 # MISSION PHASE 8 - DPS COLD SOAK
 # MP9-DPS 1 BURN
-## Page 3
+
+## Page 2
 # MISSION PHASE 11 - DPS2/FITH/APS1
 # MISSION PHASE 13 - APS2
-# MISSION PHASE 16 - RCS COLD SOAK
+# MISSION PHASE 16 - RCS COLO SOAK
 # INTEGRATION INITIALIZATION
 # ORBITAL INTEGRATION PROGRAM
 # LMP COMMAND ROUTINES
-# AS206 MISSION SCHEDULING PACKAGE
-# 206 SERVICE ROUTINES
+# AS206 MISSION SCHEOULING PACKAGE
```

Comment:
For comparison, here's the PDF on the left, and JPEG2000 image on the right:


![comparison](https://cloud.githubusercontent.com/assets/94056/26523975/44651978-42da-11e7-9752-c80e8f92975c.png)